shopfloor_product_dimension/actions/detail.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.addons.component.core import Component
+
+
+class DataDetailAction(Component):
+    _inherit = "shopfloor.data.detail.action"
+
+    @property
+    def _product_dimension_parser(self):
+        return [
+            "product_length:length",
+            "product_height:height",
+            "product_width:width",
+            "weight",
```

Comment:
@rousseldenis Why not the weight uom?

Reply:
@rousseldenis go for it, it already is a dependency for 'shopfloor' base module :)